.github/workflows/benchmarks.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           key: benches-${{ runner.os }}-unit_fluvio_protocol
 
       - name: Run Benchmarks
-        run: cargo bench -p fluvio-protocol -- --output-format bencher | tee benchmarks.txt
+        run: cargo bench -p fluvio-protocol -- --output-format bencher | tee fluvio_protocol_benches.txt
```

Comment:
Given that we have different cache keys and jobs for benchmarks I think this may not be needed. WDYT? @sehz , @morenol

Reply:
have no idea...